main.go

```diff
@@ -47,6 +48,7 @@ func init() {
 	flag.IntVar(&days, "days", 0, "No. of days to get forecast")
 	flag.IntVar(&days, "d", 0, "No. of days to get forecast (shorthand)")
 	flag.BoolVar(&ignoreAlerts, "ignore-alerts", false, "Ignore alerts in weather output")
+	flag.BoolVar(&showIcon, "show-icon", true, "Show an icon representing the weather")
```

Comment:
How about `hide-icon`? For boolean flags it feels more natural to make the "value" optional. i.e., `--hide-icon` versus `--show-icon=false`